### ISSUE TITLE
[BUG]: Klonoa 2: GIF Arbitration Bug

### ISSUE THREAD

**Issue by knight-ryu12** [PCSX2/pcsx2#5800]
### Describe the Bug

When Klonoa2 is upscaled, It will make nasty vertical lines that doesn't disappear with any Upscale hacks.

### Reproduction Steps

Run game in >=2x internal resoultion

### Expected Behavior

No lines.

### PCSX2 Revision

v1.7.2545

### Operating System

Windows 11

### CPU

Ryzen 7 5800H

### GPU

RTX 3060 M

### GS Settings

4x Internal

**Comment by kenshen112** [comment]
Can you post a screenshot of the issue? I haven't seen it on my end

**Comment by knight-ryu12** [comment]
(Pulled from Discord)

**Comment by knight-ryu12** [comment]
![image](https://user-images.githubusercontent.com/11976467/161177346-f5640ca6-ad65-4dd9-bf86-849a50ed6548.png)

**Comment by knight-ryu12** [comment]
https://github.com/PSI-Rockin/DobieStation/pull/360 issue need to be implemented

**Comment by knight-ryu12** [comment]
![image](https://user-images.githubusercontent.com/11976467/161177404-2065e343-7b6c-4f7d-aef1-b7c6069ef4bf.png)
Probably related.

**Comment by coornio** [comment]
Here's a GS dump people can run to see this issue. Seems to affect more than just Klonoa. Some trees in the first level also freak out when MTVU/IVU1 are disabled. Likely more models in later levels too.

[gs_20220511133538_Klonoa 2 - Lunatea_s Veil _NTSC-U__SLUS-20151.zip](https://github.com/PCSX2/pcsx2/files/8669143/gs_20220511133538_Klonoa.2.-.Lunatea_s.Veil._NTSC-U__SLUS-20151.zip)

![gs_20220511133538_Klonoa 2 - Lunatea_s Veil _NTSC-U__SLUS-20151](https://user-images.githubusercontent.com/11655762/167830826-51b05d9f-c9cc-4627-ba9f-f3bbdb4a00dc.png)

**Comment by refractionpcsx2** [comment]
> Here's a GS dump people can run to see this issue. Seems to affect more than just Klonoa. Some trees in the first level also freak out when MTVU/IVU1 are disabled. Likely more models in later levels too.
> 
> [gs_20220511133538_Klonoa 2 - Lunatea_s Veil _NTSC-U__SLUS-20151.zip](https://github.com/PCSX2/pcsx2/files/8669143/gs_20220511133538_Klonoa.2.-.Lunatea_s.Veil._NTSC-U__SLUS-20151.zip)
> 
> ![gs_20220511133538_Klonoa 2 - Lunatea_s Veil _NTSC-U__SLUS-20151](https://user-images.githubusercontent.com/11655762/167830826-51b05d9f-c9cc-4627-ba9f-f3bbdb4a00dc.png)

I don't see the error in that picture? anyway, it's a core problem, a GS Dump is useless.

**Comment by coornio** [comment]
Yeah, I'm aware the GS dump doesn't do much for a core issue but I thought it might be helpful for people to have a better understanding of the flickering. Here's a zoomed shot:

![gs_20220511134815](https://user-images.githubusercontent.com/11655762/167832714-179e8c3c-7452-472b-902c-1bfc28f89b09.png)

**Comment by personaone** [comment]
I just tried Klonoa in both SW and OGL mode and I was getting the green color texture in the intro as in the original screenshot and also some weird textures - so unsure what's up. Latest build.

![Klonoa 2 - Lunatea_s Veil _NTSC-U__SLUS-20151_20220822113711](https://user-images.githubusercontent.com/8123295/185890904-ae0d8c49-5f85-4447-b288-4f79cbd9fd98.png)

**Comment by refractionpcsx2** [comment]
You probably have "Instant VU" disabled.

**Comment by personaone** [comment]
> You probably have "Instant VU" disabled.

Yeah - that's right :D i turned the hack on and it fixed it :D didn't expect that!

**Comment by anakocun6** [comment]
Looks like working fine, PCSX2 v2.7.30, Vulkan, enabled: VU1 (MTVU), Vertical Sync, Mipmapping, Instant VU1 (in Advanced).

<img width="1299" height="974" alt="Image" src="https://github.com/user-attachments/assets/d62c60b8-cd48-4f64-bfc0-2ecfa7edb142" />

**Comment by Igetin** [comment]
I'm also unable to reproduce this issue using the default settings on PCSX2 v2.7.43. This is the case regardless of the renderer, and regardless of whether the "Enable Multithreaded VU1 (MTVU)" setting is enabled or not.

**Comment by refractionpcsx2** [comment]
You have to disable instant vu, so default settings or just disabling MTVU won't change anything.

**Comment by Igetin** [comment]
> You have to disable instant vu, so default settings or just disabling MTVU won't change anything.

Thanks, I was able to reproduce this on PCSX2 v2.7.43 with "Enable Instant VU1" disabled. However, for me it does not occur in the first stage (Sea of Tears), unlike what others have reported. It also seems to affect only the character models.

<table>
  <tr>
    <td><img width="200" src="https://github.com/user-attachments/assets/345cdc1f-49e2-4655-abe2-d8b8390361f1" /></td>
    <td><img width="200" src="https://github.com/user-attachments/assets/ccf162b2-449f-46d1-94d9-4ee488ce8e9c" /></td>
    <td><img width="200" src="https://github.com/user-attachments/assets/0cd93db9-8a0f-4937-bb44-d0b78e2377f0" /></td>
  </tr>
</table>
